package.json

```diff
@@ -2,9 +2,25 @@
   "name": "cross-domain-safe-weakmap",
   "version": "1.0.10",
   "description": "Cross domain safe WeakMap.",
-  "main": "index.js",
+  "main": "dist/cross-domain-safe-weakmap.js",
+  "module": "dist/module",
   "scripts": {
-    "test": "gulp test"
+    "build": "npm run babel & npm run webpack && npm run flow:build",
```

Comment:
So what do you want to do?  I don't really know why you have the shell scripts anyway...  For publishing this is what I do:

https://github.com/trainerbill/hapi-paypal/blob/master/package.json#L28-L30

I don't see much of a reason to have your publish script.  I would add a git checkout dist to make sure you never commit dist on a release and then have a release:dist that does everything without checking out dist.